entries/f/fanduel.com.json

```
@@ -0,0 +1,14 @@
+{
+  "Fanduel": {
+    "domain": "fanduel.com",
+    "url": "https://www.fanduel.com/",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://www.fanduel.com/2fa-how-it-works",
+    "keywords": [
+      "betting"
+    ]
```

Since it is US-only, a region tag would be good.
```suggestion
    ],
    "regions": [
      "us"
    ]
```